hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/pom.xml

```diff
@@ -40,6 +40,12 @@
     </dependency>
 
     <dependency>
+      <groupId>com.icegreen</groupId>
+      <artifactId>greenmail</artifactId>
+      <version>${version.com.icegreen}</version>

```

Comment:
Oh, yes.
Good catch.
I'm going to prepare a minimal fix for it and pushed to master.